src/test/java/gov/cabinetofice/gapuserservice/service/user/OneLoginUserServiceTest.java

```diff
@@ -350,5 +350,22 @@ void updateRolesShouldAddApplicantAndFindRolesWhenNoRolesPresent() {
         assertThat(updatedUser.getRoles().size()).isEqualTo(4);
         assertThat(updatedUser.getRoles().stream().anyMatch(role -> role.getName().equals(RoleEnum.APPLICANT))).isTrue();
         assertThat(updatedUser.getRoles().stream().anyMatch(role -> role.getName().equals(RoleEnum.FIND))).isTrue();
-    }}
+    }
```

Comment:
You can write more test cases to cover department being null and user having admin/super admin role(s)